### PR TITLE
add classification for fastft articles

### PR DESCRIPTION
**Pull request by aintgoin2goa**

### CODE REVIEW

**Comment by matthew-andrews** [comment]
not sure this will work…

**Comment by matthew-andrews** [comment]
oh no you're right wow, had no idea

File: server/utils/access.js [hunk 1]
```diff
@@ -39,6 +39,8 @@ module.exports = function(req, res, next) {
 						case '3' :
 							classification = 'conditional_premium';
 					}
+				}else if(article.item.location.uri.indexOf('fastft') > -1){
+					classification = 'conditional_registered';
```
Comment (matthew-andrews) [inline server/utils/access.js]: wouldn't we want this to be `conditional_standard`?

**Comment by matthew-andrews** [comment]
:+1:

File: server/utils/access.js (hunk 1 above)
Comment (aintgoin2goa) [inline server/utils/access.js]: Current ft.com barrier says "Fast FT is only available to FT.com subscribers and registered users"

**Pull request merged**